Source/RuntimeAudioImporter/Private/RuntimeAudioTranscoder.cpp

```diff
@@ -268,4 +274,8 @@ void URuntimeAudioTranscoder::TranscodeEncodedDataFromFile(const FString& FilePa
 
 		ExecuteResult(true);
 	}));
+#else
+	UE_LOG(LogRuntimeAudioImporter, Error, TEXT("file operation support disabled"));
```

Comment:
Could we add a more detailed description here? Such as: `UE_LOG(LogRuntimeAudioImporter, Error, TEXT("Unable to transcode encoded audio data from file '%s' to file '%s' because the file operation support is disabled"), *FilePathFrom, *FilePathTo);`